src/ResourceManagement/NotificationHubs/Microsoft.Azure.Management.NotificationHubs/project.json

```diff
@@ -0,0 +1,53 @@
+{
+  "version": "0.1.1-preview",
```

Comment:
this should be a major version change since you changed generators  - 1.0.0-preview